packages/grant-explorer/src/features/discovery/ExploreProjectsPage.tsx

```diff
@@ -0,0 +1,96 @@
+import { DefaultLayout } from "../common/DefaultLayout";
+import LandingHero from "./LandingHero";
+import { LandingSection } from "./LandingSection";
+import { useCartStorage } from "../../store";
+import { ApplicationStatus, CartProject } from "../api/types";
+import { useMemo, useState } from "react";
+import { ApplicationSummary } from "common/src/grantsStackDataClientContext";
+import { useApplications } from "./hooks/useApplications";
+import PaginatedProjectsList from "./PaginatedProjectsList";
+
+function createCartProjectFromApplication(
+  application: ApplicationSummary
+): CartProject {
+  return {
+    projectRegistryId: application.projectId,
+    roundId: application.roundId,
+    chainId: application.chainId,
+    grantApplicationId: createCompositeRoundApplicationId(application),
+    recipient: application.payoutWalletAddress,
+    grantApplicationFormAnswers: [],
+    status: ApplicationStatus.APPROVED,
+    applicationIndex: Number(application.roundApplicationId),
+    projectMetadata: {
+      title: application.name,
+      description: application.summaryText,
+      bannerImg: application.bannerImageCid,
+      logoImg: application.logoImageCid,
+    } as CartProject["projectMetadata"],
+    amount: "",
+  };
+}
+
+function createCompositeRoundApplicationId(application: ApplicationSummary) {
+  return `${application.roundId}-${application.roundApplicationId}`;
+}
+
+const ExploreProjectsPage = () => {
+  const [seed] = useState(() => Math.random());
```

Comment:
This will get a new seed every time the page is loaded. Could the user be confused by seeing a different order every time? By putting the seed definition two lines up (without the useState), it will use the same seed until page refresh.

Reply:
I had initially stored the seed in `sessionStorage` to make it last the session but this works as well to make it less confusing.